pkg/events/events.go

```diff
@@ -164,6 +195,31 @@ func New(ctx context.Context, name string, identifiers CombinedIdentifiers, data
 	if identifiers != nil {
 		evt.innerEvent.Identifiers = identifiers.CombinedIdentifiers().GetEntityIdentifiers()
 	}
+	authentication := &ttnpb.Event_Authentication{}
+	if p, ok := peer.FromContext(ctx); ok && p.Addr.String() != "pipe" {
+		host, _, err := net.SplitHostPort(p.Addr.String())
+		if err == nil {
```

Comment:
Nit: can shorten this to `if host, _, err := ...; err == nil`

Reply:
Disagree; prefer short `if` only for single-value returns.

Reply:
I don't think we should start discussions for nit comments. I probably shouldn't have pointed this out in the first place, and instead have let @neoaggelos write in whatever style he likes best.

FTR: My own rule of thumb is to combine it if I have max 1 value (`_` doesn't count), an optional err/ok and if the combination fits on the line.